internal/cli/cmd/info.go

```diff
@@ -20,17 +20,15 @@ import (
 // NewInfoCmd returns a new 'epinio info' command
 func NewInfoCmd(client *usercmd.EpinioClient) *cobra.Command {
 	return &cobra.Command{
-		Use:   "info",
-		Short: "Shows information about the Epinio environment",
-		Long:  "Shows status and versions for epinio's server-side components.",
+		Use:          "info",
+		Short:        "Shows information about the Epinio environment",
+		Long:         "Shows status and versions for epinio's server-side components.",
+		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cmd.SilenceUsage = true
```

Comment:
See above, `client-sync`. 

Just confirmed that `info` can currently be invoked with any number of arguments, which are ignored.